laravel-app/public/assets/js/likeUnlike.js

```diff
@@ -48,21 +45,16 @@ function handleLikeUnlike(postId, action, element) {
     const likeButton = $(`#saveLike[data-post-id="${postId}"][data-action="like"]`);
     const unlikeButton = $(`#saveUnlike[data-post-id="${postId}"][data-action="unlike"]`);
 
+    if (action === 'unlike') {
+        _likeCounter.text(parseInt(_likeCounter.text()) - 1);
+        _likedCounter.text(parseInt(_likedCounter.text()) - 1);
+        likeButton.show();
+        unlikeButton.hide();
+        return;
+    }
     
-    setTimeout(function () {
-        if (action === 'unlike') {
-            _likeCounter.text(parseInt(_likeCounter.text()) - 1);
-            _likedCounter.text(parseInt(_likedCounter.text()) - 1);
-            likeButton.show();
-            unlikeButton.hide();
-        } else {
-            _likeCounter.text(parseInt(_likeCounter.text()) + 1);
-            _likedCounter.text(parseInt(_likedCounter.text()) + 1);
-            likeButton.hide();
-            unlikeButton.show();
-        }
-
-        element.prop('disabled', false);
-        
-    }, 400);
+    _likeCounter.text(parseInt(_likeCounter.text()) + 1);
```

Comment:
whats is the difference between `_likeCounter` to `_likedcounter`

Reply:
I used that sir since I have separate button for like and unlike. The likeCounter for post without like of authUser, while likedCounter for posts already liked by the authUser.